src/AWS.Deploy.Recipes/RecipeDefinitions/ASP.NETAppAppRunner.recipe

```diff
@@ -53,11 +53,43 @@
         }
 
     ],
-
+    "Categories": [
+        {
+            "Id": "General",
+            "DisplayName": "General",
+            "Order": 10
+        },
+        {
+            "Id": "Compute",
+            "DisplayName": "Compute",
+            "Order": 20
+        },
+        {
+            "Id": "Health",
+            "DisplayName": "Health",
+            "Order": 30
+        },
+        {
+            "Id": "Permissions",
+            "DisplayName": "Permissions",
+            "Order": 40
+        },
+        {
+            "Id": "VPC",
+            "DisplayName": "VPC",
```

Comment:
nit: Thinking about if the deployment tool is someone's intro to AWS - wonder if we should spell "VPC" out in the `DisplayName`. My original instinct was to to lump this with "Networking" too.

Reply:
I debated about VPC or "Virtual Private Cloud". Mostly I see it referred in the console as VPC and I'm betting if you are completely new to AWS "Virtual Private Cloud" is still ambiguous so I opted for a short name.

Given that VPC config is complex enough on their own I would rather keep it as its own category. But if we get feedback that it would be better under networking we can make that change easy enough.